packages/react-instantsearch/src/components/Breadcrumb.js

```diff
@@ -0,0 +1,92 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import Link from './Link';
+import classNames from './classNames.js';
+import translatable from '../core/translatable';
+
+const cx = classNames('Breadcrumb');
+
+const itemsPropType = PropTypes.arrayOf(
+  PropTypes.shape({
+    label: PropTypes.string.isRequired,
+    value: PropTypes.string.isRequired,
+  })
+);
+
+class Breadcrumb extends Component {
+  static propTypes = {
+    canRefine: PropTypes.bool.isRequired,
+    createURL: PropTypes.func.isRequired,
+    items: itemsPropType,
```

Comment:
Not a proptypes expert, but this should be required too right, since there needs to be an array, with at least one item.